data/mods/ssb/abilities.ts

```diff
@@ -65,6 +65,25 @@ export const BattleAbilities: {[k: string]: ModdedAbilityData} = {
 		},
 	},
 
+	//n10siT
+	greedymagician: {
+		desc: "This Pokemon steals the item off a Pokemon it hits with an attack. Does not affect Doom Desire and Future Sight.",
```

Comment:
Another effect of this ability is that if the user already has an item, it swaps.